lib/services/robot_notifications_listener.dart

```diff
@@ -50,6 +50,9 @@ class RobotNotificationsListener {
     }
 
     Map<String, dynamic> data;
+    int displayTime = 3;
```

Comment:
Maybe store this as a duration? At this point it's all in dart so the duration could be passed right into the callback. It's also much safer unit wise.